### ISSUE TITLE
"New version of Armcord" message incorrectly displayed on startup

### ISSUE THREAD

**Issue by MMK21Hub** [smartfrigde/armcord#109]
When launching ArmCord, the "A new version of Armcord is available" message (see screenshot) is shown for five seconds or so, and then the app actually starts. This is despite there not being a newer version available.

### Logs
```
ArmCord has been run before. Skipping setup.
true
Creating custom titlebar window.
/usr/share/discord-development/resources/app.asar/ts-out/content/splash.html
Setting up CSP unstricter...
{
  customTitlebar: true,
  channel: 'canary',
  doneSetup: true,
  armcordCSP: true,
  mods: 'cumcord'
}
[113683:0510/182055.625148:ERROR:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
[113643:0510/182055.790235:ERROR:nss_util.cc(286)] After loading Root Certs, loaded==false: NSS error code: -8018
```

### Screenshots
![Screenshot of a small window containing the text: "A new version of Armcord is available. Please update to the latest version."](https://user-images.githubusercontent.com/50421330/167686956-90526b06-f5ee-43c9-b99b-875e6883a884.png)

**Comment by smartfrigde** [comment]
Open your browser and go to this [link](https://armcord.xyz/latest.json) and send me a screenshot of what you see

**Comment by MMK21Hub** [comment]
```json
{
  "version": "3.0.5",
  "update_endpoint": "https://armcord.xyz/",
  "note": "https://discord.gg/uaW5vMY3V6",
  "md5": "3d605bd6a1447afc5e4239b44cf393a3"
}
```
![image](https://user-images.githubusercontent.com/50421330/168145131-2b0be631-38fe-420f-b3ed-c8fe4d5a777d.png)

**Comment by smartfrigde** [comment]
Please try the latest release

**Comment by MMK21Hub** [comment]
Looks like it's been fixed in 3.0.6
